src/test/scala/nl.knaw.dans.easy.deposit/servlets/RichZipInputStreamSpec.scala

```diff
@@ -36,7 +36,7 @@ class RichZipInputStreamSpec extends TestSupportFixture {
   "unzipPlainEntriesTo" should "report invalid content" in {
     managed(new ZipInputStream(new ByteArrayInputStream("Lorem ipsum est".getBytes(StandardCharsets.UTF_8)))).apply(
       _.unzipPlainEntriesTo(testDir / "dummy") should matchPattern {
-        case Failure(BadRequestException(s)) if s == "ZIP file is malformed. No entries found." =>
+        case Failure(MalformedZipException(s)) if s == "No entries found." =>
```

Comment:
This can be simplified to: 

```suggestion
        case Failure(MalformedZipException("No entries found.")) =>
```

Check other similar cases yourself, please.